packages/doltgres/content/reference/sql-support/supported-types.md

```diff
@@ -84,49 +84,68 @@ For example, `timestamp` takes in a precision, but does not enforce it.
+| uuid[]         |     ✅      |                               |
+| varbit         |     ❌      | bit varying                   |
+| varbit[]       |     ❌      | bit varying[]                 |
 | varchar        |     🟠      | character varying             |
 | varchar[]      |     🟠      | character varying[]           |
-| xml            |      ❌      |                               |
-| xml[]          |      ❌      |                               |
+| xml            |     ❌      |                               |
+| xml[]          |     ❌      |                               |
 
 ## Pseudo-Types
 
 | SQL Type Name           | Implemented |
-|:------------------------|:-----------:|
-| any                     |      ❌      |
+| :---------------------- | :---------: |
+| any                     |     ❌      |
 | anyarray                |     🟠      |
-| anycompatible           |      ❌      |
-| anycompatiblearray      |      ❌      |
-| anycompatiblemultirange |      ❌      |
-| anycompatiblenonarray   |      ❌      |
-| anycompatiblerange      |      ❌      |
-| anyelement              |      ❌      |
-| anyenum                 |      ❌      |
-| anymultirange           |      ❌      |
-| anynonarray             |      ❌      |
-| anyrange                |      ❌      |
-| cstring                 |      ❌      |
-| event_trigger           |      ❌      |
-| fdw_handler             |      ❌      |
-| index_am_handler        |      ❌      |
-| internal                |      ❌      |
-| language_handler        |      ❌      |
-| pg_ddl_command          |      ❌      |
-| record                  |      ❌      |
-| table_am_handler        |      ❌      |
-| trigger                 |      ❌      |
-| tsm_handler             |      ❌      |
+| anycompatible           |     ❌      |
+| anycompatiblearray      |     ❌      |
+| anycompatiblemultirange |     ❌      |
+| anycompatiblenonarray   |     ❌      |
+| anycompatiblerange      |     ❌      |
+| anyelement              |     ❌      |
+| anyenum                 |     ❌      |
+| anymultirange           |     ❌      |
+| anynonarray             |     ❌      |
+| anyrange                |     ❌      |
+| cstring                 |     ❌      |
+| event_trigger           |     ❌      |
+| fdw_handler             |     ❌      |
+| index_am_handler        |     ❌      |
+| internal                |     ❌      |
+| language_handler        |     ❌      |
+| pg_ddl_command          |     ❌      |
+| record                  |     ❌      |
+| table_am_handler        |     ❌      |
+| trigger                 |     ❌      |
+| tsm_handler             |     ❌      |
 | unknown                 |     🟠      |
-| void                    |      ❌      |
+| void                    |     ❌      |
+
+## OID Alias Types
+
+See detailed list in the [Postgres docs](https://www.postgresql.org/docs/current/datatype-oid.html).
+
+| SQL Type Name | Supported |
+| :------------ | :-------- |
+| oid           | ✅        |
```

Comment:
Not sure where `xid` would go, but we have support for it, both `xid` and `xid[]`.